### PR TITLE
Tutorials

### PR DESCRIPTION
**Pull request by hgngo**
### Summary

If merged this pull request will provide a guide for beginner users to run the hello_world C example with Windows HELICS installer and Visual Studio 

### Proposed changes

Changes will be made to docs/introduction
Folder with images for the guide is added - folder name: visuals 
New file: hello-world-VS.md

### CODE REVIEW

**Comment by phlptp** [comment]
Here is the link to the page https://helics--1621.org.readthedocs.build/en/1621/introduction/hello-world-VS.html

I think we need a link from the Hello_world page itself so people know this is available

File: docs/introduction/hello-world-VS.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+# Running hello_world example using HELICS windows installer + Visual Studio 2019
+
+1. Make sure that you have downloaded HELICS through the installer.
+
+  - To get to the installer you can access it through either of the following links:
+    * https://github.com/GMLC-TDC/HELICS/releases
+    * https://docs.helics.org/en/latest/installation/index.html -  where you will see this:
```
Comment (trevorhardy) [inline docs/introduction/hello-world-VS.md:7]: We're re-working the installation guide right now to make it clearer to users. This screen shot might need to be updated once that draft is in place in a few weeks. We can worry about that then, though.

Comment (trevorhardy) [reply docs/introduction/hello-world-VS.md:7]: And now that I look at preview of the rendered Markdown, I can see that we almost certainly won't need to update that image. Yeah!

**Review by trevorhardy — APPROVED** [review]

File: docs/introduction/hello-world.md [hunk 2]
```diff
@@ -86,6 +86,10 @@ $ ./hello_world_receiver
 You should see `Hello, World` printed out in the terminal where you ran
 the `hello_world_receiver`.
 
+*For a guide to run this example in Visual Studio go to this link:
+[hello-world-VS](https://helics--1621.org.readthedocs.build/en/1621/introduction/hello-world-VS.html).*
```
Comment (phlptp) [inline docs/introduction/hello-world.md:90]: This should probably be a relative link vs a link the PR generated doc, I am not sure how to generate that though right now

Comment (phlptp) [reply docs/introduction/hello-world.md:90]: Lets fix this link to something that is generated then I think we can merge this

**Review by phlptp — CHANGES_REQUESTED** [review]
Fix the link, then I think we can merge

File: docs/introduction/hello-world.md (hunk 2 above)
Comment (nightlark) [reply docs/introduction/hello-world.md:90]: Relative links to markdown files should get replaced with the correct generated link when RTD generated the html files, e.g. `[hello-world-VS](./hello-world-VS.md)`

**Review by phlptp — APPROVED** [review]

**Comment by phlptp** [comment]
@nightlark  anything else you think should be changed?

**Review by nightlark — APPROVED** [review]
Nope, image sizes were the only thing.

**Pull request merged**